cmd/convert.go

```diff
@@ -96,7 +98,8 @@ func (o *convertOptions) run(plan string) error {
 	ancestryCache := map[string]string{
 		o.project: o.ancestry,
 	}
-	assets, err := o.readPlannedAssets(ctx, plan, o.project, ancestryCache, o.offline, false, o.rootOptions.errorLogger)
+	ua := fmt.Sprintf("config-validator-tf/%s", version.BuildVersion())
```

Comment:
I have a slight preference for `userAgent` over `ua` since it's more explicit... but also, I think golang styles tend towards brevity over clarity. Do you have a preference either way?